<launch>
    <!-- select group_name: left_arm or right_arm or combined-->
    <arg name="group_name" value="combined" />
    
    <!-- resolution in m (0.1, 0.05, 0.025) -->
    <arg name="r" value="0.1" />
    <!-- theta_resolution (unsigned int) -->
    <arg name="t" value="16" />
    
    <!-- name of polygon without ending -->
    <arg name="poly_name" value="tablew213h102" />
    
    <!-- bounding box x [-1.5, 1.5]; y [-1.5, 1.5]; z [-1.0, 1.0] -->
    <arg name="x1" value="0" />
    <arg name="x2" value="3.0" />
    
    <arg name="y1" value="0" />
    <arg name="y2" value="3.0" />
    
    <arg name="z1" value="1.0" />
    <arg name="z2" value="-1.0" />
    
    <!-- display color table -->
    <arg name="colortable" value="false" />
    <!-- draw arrows in 3d -->
    <arg name="3darrowdimension" value="false" />
    
    <arg name="min_percent" value="0" />
    

	<!-- DO NOT CHANGE PARAMETERS BELOW -->
	   
	<!-- path to inverse capability map -->
    <arg name="path_name" value="$(find inverse_capability_map_generator)/maps/inv_cap_$(arg poly_name)_pr2_$(arg group_name)_$(arg r)m_$(arg t)t.icpm" />
    <!-- path to polygon -->
    <arg name="path_poly" value="$(find primitive_to_polygon)/polygons/$(arg poly_name).poly" />
    
    <arg name="c" value="-c" if="$(arg colortable)" />
    <arg name="c" value="" unless="$(arg colortable)" />
	<arg name="d" value="-d" if="$(arg 3darrowdimension)" />
	<arg name="d" value="" unless="$(arg 3darrowdimension)" />

    <node pkg="inverse_capability_map" type="inverse_capability_polygon_vis" name="inverse_capability_polygon_vis"
    	args=" -p $(arg path_name)
    	-i $(arg path_poly)
    	$(arg c)
    	$(arg d)
    	-x $(arg x1) -x $(arg x2) -y $(arg y1) -y $(arg y2) -z $(arg z1) -z $(arg z2)"
		output="screen">
		<param name="minimum_percent" type="double" value="$(arg min_percent)" />
    </node>
</launch>
